runtime/jcl/common/java_lang_invoke_MethodHandleNatives.cpp

```diff
@@ -62,6 +62,28 @@ extern "C" {
 #define MN_TRUSTED_MODE			-1
 #endif /* JAVA_SPEC_VERSION >= 16 */
 
+static bool
+isPolymorphicMHMethod(J9JavaVM *vm, J9Class *declaringClass, J9UTF8 *methodName)
+{
+	if (declaringClass == J9VMJAVALANGINVOKEMETHODHANDLE(vm)) {
+		U_8 *nameData = J9UTF8_DATA(methodName);
+		U_16 nameLength = J9UTF8_LENGTH(methodName);
+
+		if ((J9UTF8_LITERAL_EQUALS(nameData, nameLength, "invoke"))
+		|| (J9UTF8_LITERAL_EQUALS(nameData, nameLength, "invokeBasic"))
+		|| (J9UTF8_LITERAL_EQUALS(nameData, nameLength, "linkToVirtual"))
+		|| (J9UTF8_LITERAL_EQUALS(nameData, nameLength, "linkToStatic"))
+		|| (J9UTF8_LITERAL_EQUALS(nameData, nameLength, "linkToSpecial"))
+		|| (J9UTF8_LITERAL_EQUALS(nameData, nameLength, "linkToInterface"))
+		|| (J9UTF8_LITERAL_EQUALS(nameData, nameLength, "linkToNative"))
```

Comment:
removed extra brackets and fixed indentation. ref: https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#formatting-conditions.
```suggestion
		if (J9UTF8_LITERAL_EQUALS(nameData, nameLength, "invoke")
			|| J9UTF8_LITERAL_EQUALS(nameData, nameLength, "invokeBasic")
			|| J9UTF8_LITERAL_EQUALS(nameData, nameLength, "linkToVirtual")
			|| J9UTF8_LITERAL_EQUALS(nameData, nameLength, "linkToStatic")
			|| J9UTF8_LITERAL_EQUALS(nameData, nameLength, "linkToSpecial")
			|| J9UTF8_LITERAL_EQUALS(nameData, nameLength, "linkToInterface")
			|| J9UTF8_LITERAL_EQUALS(nameData, nameLength, "linkToNative")
```

Reply:
+1 to removing the extra `(` & `)`.  I'd prefer to avoid the indentation